tests/e2e/util.go

```diff
@@ -6200,3 +6200,18 @@ func getBlockDevSizeInBytes(f *framework.Framework, ns string, pod *v1.Pod, devi
 	output = strings.TrimSuffix(output, "\n")
 	return strconv.ParseInt(output, 10, 64)
 }
+
+// createSnapshotInParallel creates snapshot from a given pvc
```

Comment:
"from a given pvc" -> "for the given pvc"
"a particular namespace" -> "given namespace"

Reply:
Done